tests/unit_tests/c_wrappers/res/job_queue/test_ert_qstat_proxy.py

```diff
@@ -91,7 +93,9 @@ def test_proxyfile_not_exists(tmpdir, monkeypatch):
             EXAMPLE_QSTAT_CONTENT.splitlines()
         )
     else:
-        assert len(result.stdout.splitlines()) == 5
+        lines = result.stdout.splitlines()
+        assert lines[0].decode("utf-8") == "Job Id: 15399.s034-lcam"
+        assert len(lines) >= 5
```

Comment:
Why >= 5?

Reply:
Since qstat_proxy.sh now allows itself to give garbage lines (that means irrelevant lines for other jobs) it makes less sense to be precise on the number of lines.

Reply:
Right, so that's the `grep` statement - now I got it I think :+1: